dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -230,6 +230,30 @@ done; \
 (exit $exit_code)
 {{- end }}
 
+# Journald input
+{{- if (eq .BeatName "filebeat") }}
+
+    {{- if (contains .from "wolfi") }}
+    RUN for iter in {1..10}; do \
+            apk update && \
+            apk add --no-interactive --no-progress --no-cache systemd && \
```

Comment:
This is going to have the side effect of exposing us to CVEs in the entirety of systemd which might turn out to be a significant problem. We probably want to keep this out of the wolfi image whose point is to minimize exposure to OS based CVEs.

In general it would actually be better to build only what we need or find a way to include just journalctl and nothing else but I'm not sure how possible that is.

Reply:
https://github.com/elastic/beats/pull/44108